bundles/org.eclipse.ui.themes/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.ui.themes;singleton:=true
-Bundle-Version: 1.2.2200.qualifier
```

Comment:
mh. I just see that the old version was indeed "2200". I never saw a service-qualifier that has for digits.
So "1.2.2300" but in deed be correct.
Let's see what the automated build says.

Reply:
Changed back to 1.2.2300, with 1.2.300 the builds failed